app/templates/briefs/not_is_supplier_eligible_for_brief_error.html

```diff
@@ -1,43 +1,27 @@
 {% extends "_base_page.html" %}
 
-{% block page_title %}Not eligible for brief – Digital Marketplace{% endblock %}
+{% block page_title %}Not eligible for opportunity – Digital Marketplace{% endblock %}
 
 {% block main_content %}
 
-{% if clarification_question %}
+{% with inhibited_action="ask a question about" if clarification_question else "apply for" %}
 <div class="grid-row">
   <div class="column-two-thirds">
     <header class="page-heading-smaller">
-      <h1>You can’t ask a question about this opportunity</h1>
+      <h1>You can’t {{ inhibited_action }} this opportunity</h1>
     </header>
-    {% if on_framework %}
-      <p>
-        You can’t ask a question about this opportunity because you don’t provide
-        {{ "this digital specialist role" if lot == 'digital-specialists' else "a service in this category" }}
-        at this location.
-      </p>
-    {% else %}
-      <p>You can’t ask a question about this opportunity because you’re not a {{ framework_name }} supplier.</p>
-    {% endif %}
+    <p>
+      You can’t {{ inhibited_action }} this opportunity because
+      {% if has_framework_service %}
+        you didn’t say you could provide
+        {{ "this specialist role" if has_framework_lot_service else "services in this category" }}
```

Comment:
Yes I think that's the thing - I'm pretty sure it's doing the right thing in terms of which message to show, but it's not clear from the code why.  I think the old version was marginally more obvious, but still relatively obscure. In both cases there is an assumption that if the user has reached this page then they have already failed the check for eligibility, so there isn't a need to do the full eligibility check again.  That's why the old test for `lot == digital-specialists` was enough to know that they don't provide the right specialist role - because we already know that the test here has failed: https://github.com/alphagov/digitalmarketplace-api/blob/master/app/brief_utils.py#L33

I think this is somewhere where a well-placed comment will help anyone coming back to this code in future.